com.gitlab.jakobdev.jdnbtexplorer.yml

```diff
@@ -0,0 +1,37 @@
+app-id: com.gitlab.jakobdev.jdnbtexplorer
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: jdNBTExplorer
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+
+modules:
+  - pyqt6.yml
+  - python3-modules.json
+
+  - name: jdNBTExplorer
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-deps --no-use-pep517 --prefix=$FLATPAK_DEST .
+      - install -Dm644 jdNBTExplorer/Logo.png /app/share/icons/hicolor/256x256/apps/com.gitlab.jakobdev.jdnbtexplorer.png
+      - install -Dm644 com.gitlab.jakobdev.jdnbtexplorer.desktop /app/share/applications/com.gitlab.jakobdev.jdnbtexplorer.desktop
+      - install -Dm644 com.gitlab.jakobdev.jdnbtexplorer.metainfo.xml /app/share/appdata/com.gitlab.jakobdev.jdnbtexplorer.metainfo.xml
+      - install -Dm644 Icon128.png /app/share/icons/hicolor/128x128/apps/com.gitlab.jakobdev.jdnbtexplorer.png
+      - install -Dm644 Icon128.png /app/share/app-info/icons/flatpak/128x128/com.gitlab.jakobdev.jdnbtexplorer.png
```

Comment:
```suggestion
```

Reply:
But how can I make Flathub find that Icon?

Reply:
it is handled by flatpak-builder.

Reply:
I is not found when putting it in /app/share/icons, so in which folder should I put it then, so flatpak-builder can find it?